packages/contracts-bedrock/src/periphery/drippie/Drippie.sol

```diff
@@ -244,4 +244,11 @@ contract Drippie is AssetReceiver {
 
         emit DripExecuted(_name, _name, msg.sender, block.timestamp);
     }
+
+    /// @notice Returns the status of a given drip.
+    /// @param _name Drip to check.
+    /// @return Dripstatus of the given drip.
+    function getDripStatus(string calldata _name) public view returns (DripStatus) {
```

Comment:
I added this because the compiler kept throwing `stack too deep` errors when I would try to read the status off of the `drips` mapping from the `DeployPeriphery` contract.

Reply:
I did something similar in the tests